Automattic/wp-calypso
# Hosting Command Palette: Rank current site at the top

Fixes https://github.com/Automattic/dotcom-forge/issues/4710

## Proposed Changes

Puts the current site at the top of the site list when using the command palette within an existing site context.

![CleanShot 2023-11-30 at 14 48 49@2x](https://github.com/Automattic/wp-calypso/assets/36432/8c80acc1-a788-4106-86a7-5668525b1ad1)

## Testing Instructions

1. Navigate to `/sites`.
2. Open the command palette, select 'Manage hosting configuration', and verify the site list matches your expected site order.
3. Navigate to Hosting Configuration for one of your sites.
4. Open the command palette, select 'Manage writing settings', and verify the current site appears at the top of the site list.
